keps/sig-apps/3973-consider-terminating-pods-deployment/README.md

```diff
@@ -256,6 +279,101 @@ To satisfy the requirement for tracking terminating pods, and for implementation
 we propose a new field `.status.terminatingReplicas` to the ReplicaSet's and Deployment's
 status.
 
+### Deployment Scaling Changes and a New Annotation for ReplicaSets
```

Comment:
Have you considered either re-using the deployment status fields you're adding or adding them to track that information, rather than introducing annotations?